### PR TITLE
chore: update eslint, use local eslint config

### PR DESCRIPTION
**Pull request by Jl14Salvador**

### CODE REVIEW

**Comment by forstermatth** [comment]
@Jl14Salvador @trs Since this is a public repo, we cannot use our private scripts / configs to set the CI up. It will have to remain local.

**Comment by forstermatth** [comment]
My mistake, dotfiles is also public.

File: package.json [hunk 1]
```diff
@@ -32,8 +32,8 @@
     "test:unit": "chokidar 'lib/**/*.js' 'test/**/*.js' -c 'npm run test:unit:once' --initial --silent",
     "test:unit:once": "cross-env NODE_ENV=test istanbul cover --config config/testUnit/istanbul.js _mocha -- --opts config/testUnit/mocha.opts",
     "verify": "npm run verify:js --silent",
-    "verify:js": "eslint -c config/verify/.eslintrc \"lib/**/*.js\" \"test/**/*.js\" \"config/**/*.js\" && echo ✅ verify:js success",
-    "verify:js:fix": "eslint --fix -c config/verify/.eslintrc \"lib/**/*.js\" \"test/**/*.js\" \"config/**/*.js\" && echo ✅ verify:js:fix success",
+    "verify:js": "eslint -c $HOME/.eslintrc \"lib/**/*.js\" \"test/**/*.js\" \"config/**/*.js\" && echo ✅ verify:js success",
```
Comment (forstermatth) [inline package.json]: This may not be the same file as we use on other peoples computers.

Comment (forstermatth) [reply package.json]: But I guess it's probably ok, as our file will still be used for CI.

**Comment by trs** [comment]
Yeah, this is a hard considering it's public. Those scripts demand a certain environment which we can't ensure with another's setup. But for CI purposes, it's the correct way to have it.

**Comment by forstermatth** [comment]
@Jl14Salvador There is no top-level property `message` in the eslint file in dotfiles, something else is going wrong with either how you are getting the file or otherwise.

**Comment by Jl14Salvador** [comment]
@forstermatth  so it seems as though there are bad credentials for accessing that file? i thought `chmod 777 get_file.sh ` would allow permission to see it. Maybe its just not receiving it from the dotfiles for some reason? Is there a dev ops procedure that needs to be done first?  If you see, i called cat $HOME/.eslintrc to see what the eslintrc file is showing up as in circle.

**Comment by forstermatth** [comment]
@Jl14Salvador The get file script (which you added in this PR) requires the environment variable `GITHUB_TOKEN`, which doesn't exist in the variables for this repo in circle yet. You can import this in circle from any other repo that has a variable of the same name.

**Comment by Jl14Salvador** [comment]
@forstermatth  it looks like the github token that did exist in the repo must've been outdated because it was different from the ones used in our other APIs, i just updated it.

**Pull request merged**